### PR TITLE
Education Cloud Settings

### PR DESCRIPTION
**Pull request by MatthewBlanski**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

**Review by skosgi1 — CHANGES_REQUESTED** [review]
Looks like we are missing translation files for custom labels, and a few minor changes/questions requested on Custom Labels. Functionality looks great!

File: force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 1]
```diff
@@ -3359,6 +3550,23 @@
         <shortDescription>Error msg used throughout Settings</shortDescription>
         <value>Error</value>
     </labels>
+
+    <labels>
+        <fullName>stgTrailheadTitle</fullName>
```
[the remark this answers is not in the document]
Comment (MatthewBlanski) [reply force-app/main/default/labels/CustomLabels.labels-meta.xml:3555]: These are deliberately different.

**Review by skosgi1 — APPROVED** [review]
Approved.

File: force-app/main/default/translations/ca.translation-meta.xml [hunk 2]
```diff
@@ -1,5 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Translations xmlns="http://soap.sforce.com/2006/04/metadata">
+    <customApplications>
```
Comment (skosgi1) [inline force-app/main/default/translations/ca.translation-meta.xml:3]: Looks like it pulled the unmanaged tabs here too on translation files.

**Pull request merged**